modules/mono/glue/runtime_interop.cpp

```diff
@@ -1451,6 +1465,7 @@ static const void *unmanaged_callbacks[]{
 	(void *)godotsharp_stack_info_vector_resize,
 	(void *)godotsharp_stack_info_vector_destroy,
 	(void *)godotsharp_internal_editor_file_system_update_file,
```

Comment:
Seems like the `godotsharp_internal_editor_file_system_update_file` method is now unused, so we can probably remove it.